src/BaGet.UI/src/DisplayPackage.tsx

```diff
@@ -105,6 +118,8 @@ class DisplayPackage extends React.Component<IDisplayPackageProps, IDisplayPacka
           version: entry.catalogEntry.version,
         });
 
+        dependencyGroups.push(...entry.catalogEntry.dependencyGroups);
```

Comment:
This grabs all dependencies for all versions of a package. I would only grab the dependency groups for the latest package (inside the `if (latestItem) { ... }` on line 128)

Reply:
Ah. I was trying to wrap my head around that part of the code - so I imagined that this issue would crop up, but I saw a `[0]` somewhere that led me to believe that I could assume only one CatalogEntry.